### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jdifek**
- [DEMO LINK](https://jdifek.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://jdifek.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,52 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#HOME">
+        <img
+      class="logo"
+      src="images/logo.png"
+      alt="logo">
```
Comment (lerastarynets) [inline src/index.html:20]: attrCheklist

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Almost there, just one problem left)

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
Please read carefully [style guide](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

And fix code style. now it is formated wrong. The previous comment didn't fixed.

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,55 @@
...
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#HOME">
+        <img
+      class="logo"
+      src="images/logo.png"
+      alt="logo"
+      >
+      </a>
```
Comment (Viktor-Kost) [inline src/index.html:22]: code styling fixes needed
```suggestion
      <a href="#HOME">
        <img
          class="logo"
          src="images/logo.png"
          alt="logo"
        >
      </a>
```

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,55 @@
...
+            <a class="nav_link" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Laptops & Computers"
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```
Comment (Viktor-Kost) [inline src/index.html:42]:
```suggestion
            <a
              class="nav_link"
              href="#laptops_computers"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,83 @@
+:root {
+  --main-color: #3498db;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
```
Comment (Viktor-Kost) [inline src/style.css:9]: it's not a good practice to set this rule to the full page. set only for links

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,83 @@
+:root {
+  --main-color: #3498db;
+  --header-height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:3]: it's not a good practice to hardcode height for elements with content inside. use padding for your links instead

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,83 @@
...
+  align-items:center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav_item .is-active {
+  position: relative;
+  margin-left: 0;
+  color: #00ACDC;
```
Comment (Viktor-Kost) [inline src/style.css:23]: repeated colors to variables

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,83 @@
...
+}
+
+.header {
```
Comment (Viktor-Kost) [inline src/style.css:26]: add `padding: 0 50px` to avoid using margin for `img` and `nav`

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,83 @@
...
+  align-items: center;
+  height: var(--header-height);
```
Comment (Viktor-Kost) [inline src/style.css:28]:
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,83 @@
...
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+img {
+  display: flex;
+  align-items:center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```
Comment (Viktor-Kost) [inline src/style.css:17]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,83 @@
...
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+img {
```
Comment (Viktor-Kost) [inline src/style.css:12]: style class selectors not tag ones

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,83 @@
...
+  height: var(--header-height);
+  display: flex;
+  background: #FFF;
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+}
+
+li {
+  margin-left: 20px;
+}
```
Comment (Viktor-Kost) [inline src/style.css:37]:
```suggestion
```

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,83 @@
...
+  display: inline-block;
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  line-height: 16px;
+  padding-bottom: 22px;
+  padding-top: 22px;
+}
+
+a:hover {
```
Comment (Viktor-Kost) [inline src/style.css:70]: style class selectors not tag ones

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,83 @@
...
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: "";
+  background: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:76]: variables

File: src/style.css [hunk 14]
```diff
@@ -1,3 +1,83 @@
...
+  display: flex;
+  align-items:center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav_item .is-active {
+  position: relative;
+  margin-left: 0;
```
Comment (Viktor-Kost) [inline src/style.css:22]:
```suggestion
.is-active {
  position: relative;
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 15]
```diff
@@ -1,3 +1,78 @@
...
+  background: var(--border-color);
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.nav_link[data-qa="hover"]:hover {
```
Comment (volodymyr-soltys97) [inline src/style.css:71]: No need to use attributes as css selector

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Pull request closed without merging**